### PR TITLE
Build for Windows

### PR DESCRIPTION
**Pull request by saraedum**
cc: @videlec @slel

### CODE REVIEW

**Comment by saraedum** [comment]
@isuruf: I copied some stuff over from the m4ri recipe. Let's see how this goes.

**Comment by saraedum** [comment]
@conda-forge-admin please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+chmod +x configure
+
+case `uname` in
+    Darwin|Linux)
+        export CFLAGS="-O3 -g -fPIC $CFLAGS"
+        ./configure --prefix=$PREFIX --with-e-antic=$PREFIX --with-nauty=$PREFIX --with-flint=$PREFIX --with-gmp=$PREFIX
+        ;;
+    MINGW*)
+        export PATH="$PREFIX/Library/bin:$BUILD_PREFIX/Library/bin:$RECIPE_DIR:$PATH"
+        export CC=cl_wrapper.sh
```
Comment (isuruf) [inline recipe/build.sh:17]: You need this file from `m4ri-feedstock` and also the `bld.bat`

Comment (saraedum) [reply recipe/build.sh:17]: Right. Forgot to add them to the index.

**Comment by saraedum** [comment]
@isuruf: So why would the m4ri recipe install clangdev if it's then just using the builtin compilers through the cccl wrapper?

**Comment by isuruf** [comment]
> So why would the m4ri recipe install clangdev if it's then just using the builtin compilers through the cccl wrapper?

Is it? It's calling `clang-cl`

**Comment by saraedum** [comment]
It's setting `CC=cl_wrapper.sh`.

**Comment by saraedum** [comment]
oh, and that's calling CCCL agian.

**Comment by saraedum** [comment]
@isuruf: So, if we're using clang anyway why do we need to go through the cccl wrapper? It's says that this is a wrapper for the MS compiler.

**Comment by isuruf** [comment]
Because autotools doesn't like `clang.exe`'s GNU interface on windows. (autotools would call `clang.exe` with `/EXPORT` options which `clang.exe` doesn't understand)

Basically autotools calls the compiler with GNU like syntax and MSVC like syntax.
So, we need a wrapper to convert GNU like options to MSVC like options or the other way. I found `cccl` could do the first.

**Comment by isuruf** [comment]
> It's says that this is a wrapper for the MS compiler.

True, but my modifications make it work with clang-cl.

**Comment by saraedum** [comment]
So, we need to disable VS2008 here, right?

**Comment by isuruf** [comment]
Yes, `skip: True  # [win and vc<14]`

**Comment by saraedum** [comment]
@conda-forge-admin please rerender

**Comment by saraedum** [comment]
`LINK : fatal error LNK1181: cannot open input file 'pthread.lib'`

**Comment by isuruf** [comment]
Use https://anaconda.org/conda-forge/pthreads-win32

**Comment by saraedum** [comment]
The failing `make check` is now https://github.com/Normaliz/Normaliz/issues/258.

**Comment by saraedum** [comment]
@isuruf any clue what's going on here?

**Pull request merged**